tests/plugins/test_resolve_composes.py

```diff
@@ -1118,6 +1118,55 @@ def test_only_pulp_repos(self, workflow):
             .and_return(ODCS_COMPOSE))
         self.run_plugin_with_args(workflow)
 
+    @pytest.mark.parametrize(('content_sets', 'build_only_content_sets'), (
+        (True, True),
+        (True, False),
+        (False, True),
+        (False, False),
+    ))
+    def test_only_content_sets(self, workflow, content_sets, build_only_content_sets):
+        main_cs_list = ['pulp-spam-rpms', 'pulp-bacon-rpms', 'pulp-eggs-rpms']
+        dev_cs_list = ['dev-spam-rpms', 'dev-bacon-rpms', 'dev-eggs-rpms', 'pulp-spam-rpms']
```

Comment:
How about rename it to `build_only_cs_list`?

Reply:
renamed